specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -776,6 +776,21 @@
         }
       }
     },
+    "OrcSource": {
+      "description": "A copy activity ORC source.",
```

Comment:
One more thing is that  where `OrcSource`  and `OrcSink` are used in your swagger file.

Reply:
It is the similiar usage.
Take Source as an example:
In specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json
"OrcSource" is to add a new type for "CopySource" with the defination "discriminator" in copy source(line 914). Then "CopySource" is referenced by "CopyActivityTypeProperties"(line 885)=>"CopyActivity"(line 859)=>extend a type for "Activity"(line 21)=>"Pipeline"(line 9)

In specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json
"Pipeline" was reference by "PipelineResource"(line 4106)=>used in several API.

Sink is almost the same as Source.

Reply:
Why do you use x-ms-discriminator-value in "OrcDataset" not in "OrcSource"?

Reply:
Because the real value we want for customer to use is "Orc" but not "OrcDataset". 
@Juliehzl  It will be very helpful if we can merge this PR before tomorrow, thanks.

Reply:
@TinaHu1 Thanks for your detailed explanation. Just  ensure everything is in the right way. 😊